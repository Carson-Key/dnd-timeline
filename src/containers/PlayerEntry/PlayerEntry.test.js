import React from "react"
import PlayerEntry from "./index.js"
import { render } from 'enzyme'
import toJson from "enzyme-to-json";

describe('Render PlayerEntry ->', () => {

  test('with no props', () => {
    const playerEntry = render(
      <PlayerEntry />,
    )

    expect(toJson(playerEntry)).toMatchSnapshot()
  })

})
