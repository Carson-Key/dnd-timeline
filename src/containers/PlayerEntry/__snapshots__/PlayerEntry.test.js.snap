// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PlayerEntry -> with no props 1`] = `
<div
  class="container"
>
  <br />
  <div
    class="row"
  >
    <div
      class="col"
    >
      <div
        class="jumbotron jumbotron-fluid"
      >
        <div
          class="container-fluid"
        >
          <center>
            <h1
              class="display-3"
            >
              Timeline
            </h1>
          </center>
        </div>
      </div>
    </div>
  </div>
  <div
    class="row"
  >
    <div
      class="col"
    >
      <label
        for="playerName"
      >
        What is Your Name:
      </label>
      <center>
        <div
          class="input-group"
        >
          <input
            class="form-control"
            id="playerName"
            placeholder="John Smith"
            type="text"
          />
          <div
            class="input-group-append"
          >
            <button
              class="btn btn-secondary"
            >
              Enter
            </button>
          </div>
        </div>
      </center>
    </div>
  </div>
</div>
`;
